### PR TITLE
Create a release GitHub workflow

### PR DESCRIPTION
**Pull request by DeMoorJasper**
# ↪️ Pull Request

This PR adds 3 new scripts to the root package.json.

Tag Scripts, used for updating version tag in all packages:

`yarn tag:prerelease <new-version>` for example `yarn tag:prerelease 2.0.0-beta.1.0`
`yarn tag:release <new-version>` for example `yarn tag:release 2.0.0`

Release Script: `yarn release`, runs automatically once a github version gets published

Workflow: `tag release` => `open PR` => `Merge to v2` => `Publish a GitHub release` => Published to NPM 🎉

Tested in my `parcel-plugin-svelte` repo: https://github.com/DeMoorJasper/parcel-plugin-svelte/actions/runs/127680315

### CODE REVIEW

File: .github/workflows/tag-release.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: tag-release
```
Comment (DeMoorJasper) [inline .github/workflows/tag-release.yml:1]: This workflow should automatically run whenever a release gets published

File: package.json [hunk 2]
```diff
@@ -27,7 +27,9 @@
     "test": "yarn test:unit && yarn test:integration",
     "update-readme-toc": "doctoc README.md",
     "nightly:release": "lerna publish -y --canary --preid nightly --dist-tag=nightly --exact --force-publish=*",
-    "alpha:release": "lerna publish --dist-tag=next --exact --force-publish=*"
+    "tag:prerelease": "lerna version --exact --force-publish=* --no-git-tag-version",
+    "tag:release": "lerna version --force-publish=* --no-git-tag-version",
+    "release": "lerna publish from-package --dist-tag=next"
```
Comment (DeMoorJasper) [inline package.json:32]: This script should automatically release any unreleased packages

File: package.json [hunk 3]
```diff
@@ -27,7 +27,9 @@
     "test": "yarn test:unit && yarn test:integration",
     "update-readme-toc": "doctoc README.md",
     "nightly:release": "lerna publish -y --canary --preid nightly --dist-tag=nightly --exact --force-publish=*",
-    "alpha:release": "lerna publish --dist-tag=next --exact --force-publish=*"
+    "tag:prerelease": "lerna version --exact --force-publish=* --no-git-tag-version",
+    "tag:release": "lerna version --force-publish=* --no-git-tag-version",
```
Comment (DeMoorJasper) [inline package.json:31]: These scripts automatically tag all packages with a provided version

**Comment by parcel-benchmark** [comment]
## Benchmark Results
<details><summary>Kitchen Sink ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 5.92s | +26.00ms |
| Cached | 2.46s | +371.00ms ⚠️ |

#### Cold Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/legacy/parcel.d5807e82.webp | 102.94kb | +0.00b | 221.00ms | -78.00ms 🚀 |
| dist/modern/parcel.d5807e82.webp | 102.94kb | +0.00b | 220.00ms | -79.00ms 🚀 |
| dist/legacy/index.html | 709.00b | +0.00b | 642.00ms | +58.00ms ⚠️ |
| dist/modern/index.html | 709.00b | +0.00b | 639.00ms | +57.00ms ⚠️ |
| dist/legacy/styles.afb8e31a.css | 78.00b | +0.00b | 1.53s | +267.00ms ⚠️ |
| dist/modern/styles.cda8c91b.css | 78.00b | +0.00b | 1.32s | -82.00ms 🚀 |

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/legacy/parcel.d5807e82.webp | 102.94kb | +0.00b | 272.00ms | +91.00ms ⚠️ |
| dist/modern/parcel.d5807e82.webp | 102.94kb | +0.00b | 271.00ms | +91.00ms ⚠️ |
| dist/legacy/kitchen-sink.679578a5.js | 1.16kb | +0.00b | 489.00ms | +64.00ms ⚠️ |
| dist/modern/kitchen-sink.90de237b.js | 1.16kb | +0.00b | 488.00ms | +64.00ms ⚠️ |
| dist/legacy/index.html | 709.00b | +0.00b | 490.00ms | +64.00ms ⚠️ |
| dist/modern/index.html | 709.00b | +0.00b | 488.00ms | +64.00ms ⚠️ |
| dist/legacy/styles.afb8e31a.css | 78.00b | +0.00b | 489.00ms | +64.00ms ⚠️ |
| dist/modern/styles.cda8c91b.css | 78.00b | +0.00b | 488.00ms | +64.00ms ⚠️ |

</p></details>

<details><summary>React HackerNews ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 26.36s | -3.52s 🚀 |
| Cached | 2.29s | -147.00ms 🚀 |

#### Cold Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/PermalinkedComment.89fe2d7f.js | 4.61kb | +0.00b | 11.39s | +5.47s ⚠️ |
| dist/UserProfile.8cce6c4b.js | 1.83kb | +0.00b | 11.39s | +5.47s ⚠️ |
| dist/NotFound.98e75551.js | 468.00b | +0.00b | 11.39s | +5.47s ⚠️ |
| dist/logo.24c8bf9e.png | 274.00b | +0.00b | 115.00ms | -54.00ms 🚀 |

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/index.js | 478.42kb | +0.00b | 253.00ms | -107.00ms 🚀 |
| dist/PermalinkedComment.89fe2d7f.js | 4.61kb | +0.00b | 251.00ms | -107.00ms 🚀 |
| dist/UserProfile.8cce6c4b.js | 1.83kb | +0.00b | 251.00ms | -107.00ms 🚀 |
| dist/NotFound.98e75551.js | 468.00b | +0.00b | 251.00ms | -107.00ms 🚀 |
| dist/logo.24c8bf9e.png | 274.00b | +0.00b | 196.00ms | -53.00ms 🚀 |

</p></details>

<details><summary>AtlasKit Editor ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 2.82m | -2.16s |
| Cached | 5.25s | +223.00ms |

#### Cold Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/pdfRenderer.74642224.js | 1.11mb | +0.00b | 55.59s | -4.27s 🚀 |
| dist/popup.b8247960.js | 219.68kb | +0.00b | 58.69s | -5.34s 🚀 |
| dist/ak-editor.d34ecfd1.js | 137.49kb | +0.00b | 14.76s | -2.18s 🚀 |
| dist/EmojiPickerComponent.1b172fe0.js | 98.95kb | +0.00b | 14.86s | -2.08s 🚀 |
| dist/Toolbar.f5e5d9f1.js | 93.03kb | +0.00b | 1.02m | -4.30s 🚀 |
| dist/media-viewer.a42ca57a.js | 71.07kb | +0.00b | 18.48s | -3.23s 🚀 |
| dist/card.e4d8099d.js | 61.60kb | +0.00b | 17.01s | -3.53s 🚀 |
| dist/component.b9da2aab.js | 56.07kb | +0.00b | 5.55s | -1.72s 🚀 |
| dist/Modal.3e325a45.js | 41.56kb | +0.00b | 2.85s | -1.31s 🚀 |
| dist/component.a8512b92.js | 28.98kb | +0.00b | 5.85s | -427.00ms 🚀 |
| dist/DatePicker.d24458b4.js | 21.54kb | +0.00b | 11.10s | -1.87s 🚀 |
| dist/EmojiPickerComponent.38775e63.js | 18.33kb | +0.00b | 51.81s | +34.47s ⚠️ |
| dist/clipboard.9f0c9743.js | 17.18kb | +0.00b | 1.00m | -4.31s 🚀 |
| dist/js.baeda348.js | 16.46kb | +0.00b | 5.48s | -1.70s 🚀 |
| dist/smartMediaEditor.ccb41b38.js | 14.62kb | +0.00b | 1.00m | -5.28s 🚀 |
| dist/ui.9e4d74b0.js | 14.45kb | +0.00b | 17.00s | -1.47s 🚀 |
| dist/workerHasher.ebf0b386.js | 11.83kb | +0.00b | 14.75s | -2.19s 🚀 |
| dist/EmojiTypeAheadComponent.899f7d25.js | 9.20kb | +0.00b | 15.67s | -1.95s 🚀 |
| dist/card.40596af5.js | 6.29kb | +0.00b | 18.19s | -2.35s 🚀 |
| dist/media-viewer.2db4dc6f.js | 4.18kb | +0.00b | 18.55s | -3.17s 🚀 |
| dist/EmojiPickerComponent.bd593806.js | 3.92kb | +0.00b | 51.81s | +34.19s ⚠️ |
| dist/16.cb2f1a3a.js | 3.72kb | +0.00b | 2.85s | -1.24s 🚀 |
| dist/Modal.1111ef83.js | 3.50kb | +0.00b | 2.85s | -1.24s 🚀 |
| dist/png-chunks-extract.c0f57e67.js | 3.49kb | +0.00b | 5.48s | -1.70s 🚀 |
| dist/dropzone.79e05ece.js | 3.44kb | +0.00b | 55.59s | -3.95s 🚀 |
| dist/16.c7be7376.js | 2.54kb | +0.00b | 5.45s | -1.72s 🚀 |
| dist/EmojiUploadComponent.4946a1c6.js | 2.50kb | +0.00b | 51.83s | +34.21s ⚠️ |
| dist/ResourcedEmojiComponent.0979b683.js | 2.24kb | +0.00b | 5.77s | -411.00ms 🚀 |
| dist/card.444c6665.js | 2.19kb | +0.00b | 33.87s | +12.14s ⚠️ |
| dist/16.bb063a1f.js | 1.83kb | +0.00b | 5.86s | -1.65s 🚀 |
| dist/date.56d1aa9e.js | 1.81kb | +0.00b | 6.00s | -1.78s 🚀 |
| dist/media-card-analytics-error-boundary.2a1f7d61.js | 1.81kb | +0.00b | 17.01s | -3.53s 🚀 |
| dist/16.b1b0372d.js | 1.77kb | +0.00b | 5.63s | -2.32s 🚀 |
| dist/images.16b3af40.js | 1.75kb | +0.00b | 6.03s | -1.89s 🚀 |
| dist/feedback.b59078b6.js | 1.72kb | +0.00b | 6.61s | -1.76s 🚀 |
| dist/ak-editor.7db5aa7b.js | 1.60kb | +0.00b | 1.23m | +8.27s ⚠️ |
| dist/list-number.d2097959.js | 1.54kb | +0.00b | 6.12s | -1.95s 🚀 |
| dist/status.f61d9ac5.js | 1.54kb | +0.00b | 6.28s | -1.95s 🚀 |
| dist/16.450f3a0d.js | 1.49kb | +0.00b | 5.81s | -1.59s 🚀 |
| dist/code.4289d7dd.js | 1.46kb | +0.00b | 5.97s | -1.80s 🚀 |
| dist/16.2b801dd1.js | 1.44kb | +0.00b | 5.63s | -2.32s 🚀 |
| dist/16.fe51c023.js | 1.44kb | +0.00b | 5.65s | -2.39s 🚀 |
| dist/16.7d4b2402.js | 1.42kb | +0.00b | 5.96s | -1.74s 🚀 |
| dist/link.36b44af7.js | 1.38kb | +0.00b | 6.08s | -1.94s 🚀 |
| dist/heading6.675c185b.js | 1.38kb | +0.00b | 6.59s | -1.78s 🚀 |
| dist/16.6b0e338c.js | 1.38kb | +0.00b | 5.96s | -1.65s 🚀 |
| dist/heading3.4e88bcd6.js | 1.37kb | +0.00b | 6.33s | -1.99s 🚀 |
| dist/16.e32aa49c.js | 1.34kb | +0.00b | 2.85s | -1.24s 🚀 |
| dist/16.50f9c790.js | 1.33kb | +0.00b | 5.90s | -1.70s 🚀 |
| dist/emoji.3654aec7.js | 1.31kb | +0.00b | 6.03s | -1.89s 🚀 |
| dist/16.d1b58d7c.js | 1.29kb | +0.00b | 5.96s | -1.82s 🚀 |
| dist/16.1dbb5df4.js | 1.28kb | +0.00b | 2.85s | -1.24s 🚀 |
| dist/Modal.6509cbe3.js | 1.26kb | +0.00b | 5.45s | -725.00ms 🚀 |
| dist/heading5.e338f1e8.js | 1.25kb | +0.00b | 6.59s | -1.78s 🚀 |
| dist/16.ceac57cd.js | 1.25kb | +0.00b | 5.82s | -1.60s 🚀 |
| dist/expand.4d883914.js | 1.24kb | +0.00b | 6.73s | -2.26s 🚀 |
| dist/16.0efadc10.js | 1.23kb | +0.00b | 5.63s | -2.32s 🚀 |
| dist/16.35be54a4.js | 1.23kb | +0.00b | 5.72s | -1.67s 🚀 |
| dist/16.6be9bbc8.js | 1.23kb | +0.00b | 5.81s | -1.59s 🚀 |
| dist/16.9015e8cf.js | 1.23kb | +0.00b | 5.92s | -1.30s 🚀 |
| dist/16.234ee133.js | 1.23kb | +0.00b | 5.77s | -412.00ms 🚀 |
| dist/heading2.89a8e126.js | 1.19kb | +0.00b | 6.31s | -1.92s 🚀 |
| dist/mention.ae379620.js | 1.15kb | +0.00b | 6.12s | -1.95s 🚀 |
| dist/heading4.b14d5ea4.js | 1.14kb | +0.00b | 6.35s | -2.02s 🚀 |
| dist/layout.d9df99ef.js | 1.13kb | +0.00b | 6.04s | -1.88s 🚀 |
| dist/divider.4476c3de.js | 1.11kb | +0.00b | 6.03s | -1.89s 🚀 |
| dist/quote.4733acb8.js | 1.10kb | +0.00b | 6.24s | -43.72s 🚀 |
| dist/panel-warning.e55309d2.js | 1.06kb | +0.00b | 6.21s | -43.74s 🚀 |
| dist/list.cf500a90.js | 1.03kb | +0.00b | 6.12s | -1.95s 🚀 |
| dist/heading1.165705a7.js | 1.03kb | +0.00b | 6.31s | -1.91s 🚀 |
| dist/panel-error.364fbf3c.js | 987.00b | +0.00b | 6.14s | -1.94s 🚀 |
| dist/panel.e4fe05d1.js | 978.00b | +0.00b | 6.22s | -1.94s 🚀 |
| dist/table.fd1280b6.js | 970.00b | +0.00b | 6.31s | -1.91s 🚀 |
| dist/panel-success.66723ab0.js | 926.00b | +0.00b | 6.21s | -43.74s 🚀 |
| dist/panel-note.a8b696c4.js | 922.00b | +0.00b | 6.18s | -43.78s 🚀 |
| dist/media-viewer-analytics-error-boundary.ee954f0e.js | 892.00b | +0.00b | 18.59s | -3.14s 🚀 |
| dist/EmojiTypeAheadComponent.441c229c.js | 619.00b | +0.00b | 15.77s | -2.70s 🚀 |
| dist/simpleHasher.dd9d4306.js | 609.00b | +0.00b | 51.81s | +34.47s ⚠️ |
| dist/index.html | 219.00b | +0.00b | 3.03s | +2.02s ⚠️ |
| dist/media-card-analytics-error-boundary.2da782a4.js | 201.00b | +0.00b | 33.88s | -3.15s 🚀 |

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/ak-editor.1970ea5c.js | 2.57mb | -4.00b 🚀 | 602.00ms | +82.00ms ⚠️ |
| dist/pdfRenderer.74642224.js | 1.11mb | +0.00b | 587.00ms | +69.00ms ⚠️ |
| dist/editorView.918fe31c.js | 586.93kb | +0.00b | 587.00ms | +68.00ms ⚠️ |
| dist/popup.b8247960.js | 219.68kb | +0.00b | 587.00ms | +68.00ms ⚠️ |
| dist/ak-editor.d34ecfd1.js | 137.49kb | +0.00b | 587.00ms | +69.00ms ⚠️ |
| dist/EmojiPickerComponent.1b172fe0.js | 98.95kb | +0.00b | 587.00ms | +69.00ms ⚠️ |
| dist/Toolbar.f5e5d9f1.js | 93.03kb | +0.00b | 587.00ms | +68.00ms ⚠️ |
| dist/media-viewer.a42ca57a.js | 71.07kb | +0.00b | 587.00ms | +69.00ms ⚠️ |
| dist/card.e4d8099d.js | 61.60kb | +0.00b | 587.00ms | +69.00ms ⚠️ |
| dist/card.f4e9ff08.js | 61.28kb | +0.00b | 587.00ms | +68.00ms ⚠️ |
| dist/component.b9da2aab.js | 56.07kb | +0.00b | 594.00ms | +75.00ms ⚠️ |
| dist/Modal.3e325a45.js | 41.56kb | +0.00b | 593.00ms | +73.00ms ⚠️ |
| dist/component.a8512b92.js | 28.98kb | +0.00b | 593.00ms | +73.00ms ⚠️ |
| dist/DatePicker.d24458b4.js | 21.54kb | +0.00b | 587.00ms | +69.00ms ⚠️ |
| dist/EmojiPickerComponent.38775e63.js | 18.33kb | +0.00b | 587.00ms | +69.00ms ⚠️ |
| dist/clipboard.9f0c9743.js | 17.18kb | +0.00b | 587.00ms | +68.00ms ⚠️ |
| dist/js.baeda348.js | 16.46kb | +0.00b | 594.00ms | +75.00ms ⚠️ |
| dist/dropzone.ddbef039.js | 14.69kb | +0.00b | 587.00ms | +68.00ms ⚠️ |
| dist/smartMediaEditor.ccb41b38.js | 14.62kb | +0.00b | 587.00ms | +68.00ms ⚠️ |
| dist/ui.9e4d74b0.js | 14.45kb | +0.00b | 587.00ms | +69.00ms ⚠️ |
| dist/workerHasher.ebf0b386.js | 11.83kb | +0.00b | 587.00ms | +69.00ms ⚠️ |
| dist/EmojiTypeAheadComponent.899f7d25.js | 9.20kb | +0.00b | 587.00ms | +69.00ms ⚠️ |
| dist/card.40596af5.js | 6.29kb | +0.00b | 587.00ms | +69.00ms ⚠️ |
| dist/media-viewer.2db4dc6f.js | 4.18kb | +0.00b | 587.00ms | +68.00ms ⚠️ |
| dist/EmojiPickerComponent.bd593806.js | 3.92kb | +0.00b | 587.00ms | +69.00ms ⚠️ |
| dist/16.cb2f1a3a.js | 3.72kb | +0.00b | 593.00ms | +73.00ms ⚠️ |
| dist/Modal.1111ef83.js | 3.50kb | +0.00b | 593.00ms | +73.00ms ⚠️ |
| dist/png-chunks-extract.c0f57e67.js | 3.49kb | +0.00b | 594.00ms | +75.00ms ⚠️ |
| dist/ak-editor.e6be06da.css | 3.46kb | +0.00b | 587.00ms | +68.00ms ⚠️ |
| dist/dropzone.79e05ece.js | 3.44kb | +0.00b | 587.00ms | +68.00ms ⚠️ |
| dist/16.c7be7376.js | 2.54kb | +0.00b | 594.00ms | +75.00ms ⚠️ |
| dist/EmojiUploadComponent.4946a1c6.js | 2.50kb | +0.00b | 587.00ms | +69.00ms ⚠️ |
| dist/ResourcedEmojiComponent.0979b683.js | 2.24kb | +0.00b | 594.00ms | +74.00ms ⚠️ |
| dist/card.444c6665.js | 2.19kb | +0.00b | 587.00ms | +68.00ms ⚠️ |
| dist/16.bb063a1f.js | 1.83kb | +0.00b | 593.00ms | +74.00ms ⚠️ |
| dist/media-picker-analytics-error-boundary.187d26a0.js | 1.82kb | +0.00b | 587.00ms | +68.00ms ⚠️ |
| dist/date.56d1aa9e.js | 1.81kb | +0.00b | 593.00ms | +74.00ms ⚠️ |
| dist/media-card-analytics-error-boundary.2a1f7d61.js | 1.81kb | +0.00b | 587.00ms | +69.00ms ⚠️ |
| dist/media-card-analytics-error-boundary.cc002bbf.js | 1.81kb | +0.00b | 587.00ms | +68.00ms ⚠️ |
| dist/browser.90c1ebeb.js | 1.79kb | +0.00b | 587.00ms | +68.00ms ⚠️ |
| dist/16.b1b0372d.js | 1.77kb | +0.00b | 594.00ms | +75.00ms ⚠️ |
| dist/images.16b3af40.js | 1.75kb | +0.00b | 588.00ms | +68.00ms ⚠️ |
| dist/feedback.b59078b6.js | 1.72kb | +0.00b | 587.00ms | +69.00ms ⚠️ |
| dist/workerHasher.9975765a.js | 1.65kb | +0.00b | 587.00ms | +69.00ms ⚠️ |
| dist/ak-editor.7db5aa7b.js | 1.60kb | +0.00b | 586.00ms | +68.00ms ⚠️ |
| dist/list-number.d2097959.js | 1.54kb | +0.00b | 588.00ms | +69.00ms ⚠️ |
| dist/status.f61d9ac5.js | 1.54kb | +0.00b | 588.00ms | +69.00ms ⚠️ |
| dist/16.450f3a0d.js | 1.49kb | +0.00b | 593.00ms | +74.00ms ⚠️ |
| dist/code.4289d7dd.js | 1.46kb | +0.00b | 593.00ms | +74.00ms ⚠️ |
| dist/16.2b801dd1.js | 1.44kb | +0.00b | 594.00ms | +75.00ms ⚠️ |
| dist/16.fe51c023.js | 1.44kb | +0.00b | 594.00ms | +75.00ms ⚠️ |
| dist/16.7d4b2402.js | 1.42kb | +0.00b | 593.00ms | +74.00ms ⚠️ |
| dist/link.36b44af7.js | 1.38kb | +0.00b | 588.00ms | +68.00ms ⚠️ |
| dist/heading6.675c185b.js | 1.38kb | +0.00b | 587.00ms | +69.00ms ⚠️ |
| dist/16.6b0e338c.js | 1.38kb | +0.00b | 593.00ms | +74.00ms ⚠️ |
| dist/heading3.4e88bcd6.js | 1.37kb | +0.00b | 588.00ms | +70.00ms ⚠️ |
| dist/16.e32aa49c.js | 1.34kb | +0.00b | 593.00ms | +73.00ms ⚠️ |
| dist/16.50f9c790.js | 1.33kb | +0.00b | 593.00ms | +74.00ms ⚠️ |
| dist/emoji.3654aec7.js | 1.31kb | +0.00b | 593.00ms | +74.00ms ⚠️ |
| dist/16.81746ce1.js | 1.29kb | +0.00b | 593.00ms | +74.00ms ⚠️ |
| dist/16.d1b58d7c.js | 1.29kb | +0.00b | 593.00ms | +74.00ms ⚠️ |
| dist/16.1dbb5df4.js | 1.28kb | +0.00b | 593.00ms | +73.00ms ⚠️ |
| dist/Modal.6509cbe3.js | 1.26kb | +0.00b | 593.00ms | +73.00ms ⚠️ |
| dist/heading5.e338f1e8.js | 1.25kb | +0.00b | 588.00ms | +70.00ms ⚠️ |
| dist/16.ceac57cd.js | 1.25kb | +0.00b | 593.00ms | +74.00ms ⚠️ |
| dist/expand.4d883914.js | 1.24kb | +0.00b | 587.00ms | +69.00ms ⚠️ |
| dist/16.0efadc10.js | 1.23kb | +0.00b | 594.00ms | +75.00ms ⚠️ |
| dist/16.35be54a4.js | 1.23kb | +0.00b | 593.00ms | +74.00ms ⚠️ |
| dist/16.6be9bbc8.js | 1.23kb | +0.00b | 593.00ms | +74.00ms ⚠️ |
| dist/16.9015e8cf.js | 1.23kb | +0.00b | 594.00ms | +75.00ms ⚠️ |
| dist/16.234ee133.js | 1.23kb | +0.00b | 593.00ms | +73.00ms ⚠️ |
| dist/heading2.89a8e126.js | 1.19kb | +0.00b | 588.00ms | +70.00ms ⚠️ |
| dist/16.7ca41ffe.js | 1.16kb | +0.00b | 593.00ms | +74.00ms ⚠️ |
| dist/mention.ae379620.js | 1.15kb | +0.00b | 588.00ms | +69.00ms ⚠️ |
| dist/heading4.b14d5ea4.js | 1.14kb | +0.00b | 588.00ms | +70.00ms ⚠️ |
| dist/layout.d9df99ef.js | 1.13kb | +0.00b | 588.00ms | +68.00ms ⚠️ |
| dist/divider.4476c3de.js | 1.11kb | +0.00b | 593.00ms | +74.00ms ⚠️ |
| dist/quote.4733acb8.js | 1.10kb | +0.00b | 588.00ms | +69.00ms ⚠️ |
| dist/action.b9409056.js | 1.08kb | +0.00b | 593.00ms | +74.00ms ⚠️ |
| dist/decision.77ee0aaa.js | 1.07kb | +0.00b | 593.00ms | +74.00ms ⚠️ |
| dist/panel-warning.e55309d2.js | 1.06kb | +0.00b | 588.00ms | +69.00ms ⚠️ |
| dist/list.cf500a90.js | 1.03kb | +0.00b | 588.00ms | +69.00ms ⚠️ |
| dist/heading1.165705a7.js | 1.03kb | +0.00b | 588.00ms | +70.00ms ⚠️ |
| dist/panel-error.364fbf3c.js | 987.00b | +0.00b | 588.00ms | +69.00ms ⚠️ |
| dist/panel.e4fe05d1.js | 978.00b | +0.00b | 588.00ms | +69.00ms ⚠️ |
| dist/table.fd1280b6.js | 970.00b | +0.00b | 588.00ms | +70.00ms ⚠️ |
| dist/panel-success.66723ab0.js | 926.00b | +0.00b | 588.00ms | +69.00ms ⚠️ |
| dist/panel-note.a8b696c4.js | 922.00b | +0.00b | 588.00ms | +69.00ms ⚠️ |
| dist/media-viewer-analytics-error-boundary.ee954f0e.js | 892.00b | +0.00b | 587.00ms | +68.00ms ⚠️ |
| dist/EmojiTypeAheadComponent.441c229c.js | 619.00b | +0.00b | 587.00ms | +69.00ms ⚠️ |
| dist/simpleHasher.dd9d4306.js | 609.00b | +0.00b | 587.00ms | +69.00ms ⚠️ |
| dist/index.html | 219.00b | +0.00b | 604.00ms | +83.00ms ⚠️ |
| dist/media-card-analytics-error-boundary.2da782a4.js | 201.00b | +0.00b | 587.00ms | +68.00ms ⚠️ |

</p></details>

<details><summary>Three.js x4 🚨</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | **FAILED** | -0.00ms |
| Cached | **FAILED** | -0.00ms |

#### Cold Bundles

*No bundles found, this is probably a failed build...*
#### Cached Bundles

*No bundles found, this is probably a failed build...*
</p></details>

**[Click here to view a detailed benchmark overview.](https://benchmark.parceljs.org/benchmark/267681485272646156)**

**Comment by DeMoorJasper** [comment]
@devongovett I’ve added a flag to the commands to ensure it does not create tags

**Pull request merged**